modules/sage/1.0/sage.smk

```diff
@@ -0,0 +1,255 @@
+        bcftools sort --max-mem {resources.mem_mb}M -Oz -o {output.vcf_passed}
+        > {log.stdout} 2> {log.stderr}
+          &&
+        tabix -p vcf {output.vcf_passed} >> {log.stdout} 2>> {log.stderr}
+        """)
+
+# Split filtered VCF file into snvs and indels
+rule _sage_split_vcf:
+    input:
+        vcf_passed = str(rules._sage_filter_vcf.output.vcf_passed)
+    output:
+        indels = CFG["dirs"]["vcf"] + "indels/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/indels.vcf.gz",
+        indels_tbi = CFG["dirs"]["vcf"] + "indels/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/indels.vcf.gz.tbi",
+        snvs = CFG["dirs"]["vcf"] + "snvs/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/snvs.vcf.gz",
+        snvs_tbi = CFG["dirs"]["vcf"] + "snvs/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/snvs.vcf.gz.tbi",        
+    log:
+        stdout = CFG["logs"]["vcf"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/split_passed.stdout.log",
+        stderr = CFG["logs"]["vcf"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/split_passed.stderr.log"
+    conda:
+        CFG["conda_envs"]["bcftools"]
+    threads:
+        CFG["threads"]["filter"]
+    resources:
+        **CFG["resources"]["filter"]
+    shell:
+        op.as_one_line("""
+        bcftools view -v indels {input.vcf_passed} -Oz -o {output.indels} > {log.stdout} 2> {log.stderr}
+          &&
+        tabix -p vcf {output.indels} >> {log.stdout} 2>> {log.stderr}
+          &&
+        bcftools view -v snps {input.vcf_passed} -Oz -o {output.snvs} >> {log.stdout} 2>> {log.stderr}
+          &&
+        tabix -p vcf {output.snvs} >> {log.stdout} 2>> {log.stderr}
+        """)
+
+
+
+# Symlinks the final output files into the module results directory (under '99-outputs/')
+rule _sage_output_vcf:
+    input:
+        combined = str(rules._sage_filter_vcf.output.vcf_passed),
+        indels = str(rules._sage_split_vcf.output.indels),
+        snvs = str(rules._sage_split_vcf.output.snvs),
+    output:
+        combined = CFG["dirs"]["outputs"] + "combined/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.combined.vcf.gz",
+        combined_tbi = CFG["dirs"]["outputs"] + "combined/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.combined.vcf.gz.tbi",
+        indels = CFG["dirs"]["outputs"] + "indels/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.indels.vcf.gz",
+        indels_tbi = CFG["dirs"]["outputs"] + "indels/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.indels.vcf.gz.tbi",
+        snvs = CFG["dirs"]["outputs"] + "snvs/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.snvs.vcf.gz",
+        snvs_tbi = CFG["dirs"]["outputs"] + "snvs/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.sage.snvs.vcf.gz.tbi"
+    run:
```

Comment:
I urge you to consider changing this to use Laura's new "in_module" feature of relative_symlink as I did in the Battenberg module. Don't change it for the inputs but do it for all outputs and any intermediate steps that make symlinks.

Reply:
I have included this option, and as well included the check for oncopipe version.